src/Parser/DeathKnight/Frost/CONFIG.js

```diff
@@ -17,16 +17,20 @@ export default {
   // If this spec's analysis does not show a complete picture please mention this in the `<Warning>` component.
   description: (
     <Wrapper>
-      Welcome to the Frost Death Knight analyzer! We hope you find these suggestions and statistics useful.<br /><br />
+      Welcome to the Frost Death Knight analyzer! This analyzer is still under construction but I hope you will find something useful.  The current plan is to have a finished analyzer by the release of Battle for Azeroth.  If you have any comments or suggestions feel free to contact Khazak(Khazak#3360) on Discord.<br /><br />
```

Comment:
"... I hope you will find something useful." sounds to me like you're implying they should find something useful elsewhere. How about replacing it with "... I hope you will find the statistics it does provide useful."